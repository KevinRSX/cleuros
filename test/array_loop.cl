MAIN()
	b := 10
	a := [0, b, 2, 3, 4, 5, 6, 7, 8, 9]
	i := 0
	while i < 10
		a[i] := a[i] * a[i]
		i := i + 1
	while i > 0
		PRINT(a[i - 1])
		i := i - 1
