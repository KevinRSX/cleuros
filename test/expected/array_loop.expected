81
64
49
36
25
16
9
4
100
0
